tools/flow9/bexp/bexp.flow

```diff
@@ -12,7 +12,7 @@ export {
 		BVar, BLet, BIf, BCall, BLambda, BSequence, BConstruct, BField, BSetMutable, BSwitch, BArray;
 
 		BVar(id : string, type : BType);
-		BLet(id : string, value : BExp, body : BExp, type : BType);
+		BLet(id : string, value : BExp, body : BExp, type : BType, supertype : BType);
```

Comment:
Explain that this supertype is of the value, which type is for the body. Maybe do it like this?

   BLet(id : string, idType : BType, value : BExp, body : BExp, bodyType : BType);